x/keyshare/types/message_submit_encrypted_keyshare_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/Fairblock/fairyring/testutil/sample"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgSubmitEncryptedKeyshare_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  MsgSubmitEncryptedKeyshare
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: MsgSubmitEncryptedKeyshare{
+				Creator: "invalid_address",
+			},
+			err: sdkerrors.ErrInvalidAddress,
+		}, {
+			name: "valid address",
+			msg: MsgSubmitEncryptedKeyshare{
+				Creator: sample.AccAddress(),
+			},
+		},
```

Comment:
Add tests for other invalid fields

Reply:
will do this in next pr (with all other tests)